api/src/main/java/org/openmrs/scheduler/TaskFactory.java

```diff
@@ -65,10 +65,8 @@ public Task createInstance(TaskDefinition taskDefinition) throws SchedulerExcept
 			throw new SchedulerException("could not load class", cnfe);
 		}
 		catch (Exception e) {
-			if (log.isDebugEnabled()) {
 				// don't need to log errors here necessarily.  If its needed, the calling method can log it.
-				log.debug("Error creating new task for class " + taskDefinition.getTaskClass(), e);
-			}
+				log.debug("Error creating new task for class {}", taskDefinition.getTaskClass(), e);
```

Comment:
```suggestion
			log.debug("Error creating new task for class {}", taskDefinition.getTaskClass(), e);
```

This is identical to @sacull's comment above.